src/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC } from "react";
+import { IPaginationProps } from "./pagination.interface";
+import { StylesWrapper } from "./pagination.styles";
+import {
+  BsChevronLeft,
+  BsChevronRight,
+  BsChevronDoubleLeft,
+  BsChevronDoubleRight,
+} from "react-icons/bs";
+
+export const Pagination: FC<IPaginationProps> = ({
+  totalPage,
+  currentPage,
+  onPageClick,
+}) => {
+  if (!totalPage || totalPage == 1) {
+    return null;
+  }
+
+  const getClassName = (id: number) => {
```

Comment:
To avoid developing a function each time for calculating the classname. You can install the classnames npm package: 
https://www.npmjs.com/package/classnames. This example would look like this:

```
classnames('page-button',  {
  'active': id + 1 === currentPage,
})
```

Reply:
🥔 I've installed this package and it worked perfectly! It's included in the latest push :)